crates/router/src/connector/checkout/transformers.rs

```diff
@@ -86,7 +87,7 @@ impl<F, T>
     ) -> Result<Self, Self::Error> {
         Ok(Self {
             response: Ok(types::PaymentsResponseData::TokenizationResponse {
-                token: item.response.token,
+                token: item.response.token.peek().to_owned(),
```

Comment:
same here, consider using `.expose()` as you are using `to_owned()` (i.e. `clone()`) regardless